### DISCUSSION TITLE
请教怎么设置网关tproxy代理？

### DISCUSSION THREAD

**Discussion by jones-76** [kkkgo/PaoPaoGateWay#31] [问和答]
设置好PPGW和PPDNS后，所有访问链接都是Socks5(tcp)模式，请问可以设置tproxy透明代理吗？
![1698341671218](https://github.com/kkkgo/PaoPaoGateWay/assets/58469169/f96ccb89-f0ef-433f-be62-b993e1220e94)

**Comment by kkkgo** [comment]
你是不是docker定制的sniff，前置嗅探的话是这样显示的，文档说明了这个

**Reply by jones-76** [reply]
感谢大佬现在还帮忙回答问题，刚发完问题又去看了下docker定制的说明，确实看见了这个问题，现在已经取消sniff嗅探参数重新生成了，谢谢！

**Reply by kkkgo** [reply]
嗅探模块不属于clash的一部分，所以无法显示在clash面板中。嗅探是tproxy的。
